packages/Webkul/Shop/src/Http/Requests/CartAddressRequest.php

```diff
@@ -58,6 +59,11 @@ private function mergeAddressRules(string $addressType)
             "{$addressType}.postcode"     => core()->isPostCodeRequired() ? ['required', new PostCode] : [new PostCode],
             "{$addressType}.phone"        => ['required', new PhoneNumber],
         ]);
+        if ($addressType == 'billing') {
+            $this->mergeWithRules([
+                "{$addressType}.vat_id"       => [new VatIdRule],
```

Comment:
Since this is a single key-value pair, the arrow indentation should be a single space.

Reply:
~~~php
"{$addressType}.vat_id" => [new VatIdRule],
~~~